solidity/contracts/system/TBTCSystem.sol

```diff
@@ -173,6 +173,7 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
     /// @notice One-time-use emergency function to disallow future deposit creation for 10 days.
     function emergencyPauseNewDeposits() external onlyOwner returns (bool) {
         require(pausedTimestamp == 0, "emergencyPauseNewDeposits can only be called once");
+        require(block.timestamp - initializedTimestamp < 365 days, "Can only be called within 365 days of initialization");
```

Comment:
We should be consistent with the previous `require` and say e.g. `emergencyPauseNewDeposits can only be called…` (or flip the other one, either way).

Reply:
I prefer including the method name, but we have a linter rule for long error messages.

Reply:
Interesting. Looks like that’s an ethlint default. Feels like we should change it...

Reply:
Addressed in https://github.com/keep-network/solium-config-keep/pull/1